### PR TITLE
Feature/caddy upgrade

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Upgrade PacketFence to caddy v1.0.3 + minor patches (https://github.com/caddyserver/caddy/commit/99914d22043f707f3f69bb5ee509d3353d75e943)

# Impacts
All golang binaries based on caddy

# Delete branch after merge
YES

# NEWS file entries
(REQUIRED, but may be optional...)
## New Features
* Upgraded to Caddy server v1.0.3 for pfhttpd (api-frontend, httpd.dispatcher, pfipset, pfsso)

### CODE REVIEW

**Comment by julsemaan** [comment]
The review should likely focus on https://github.com/inverse-inc/packetfence/pull/4857/files/77fcf8534deeb92b5d57ebdc7b093963786a8416..86174b6d94139dd149f9eb524371e20c8febfe0b which is the commits that exclude the copy/paste of the caddy source

File: go/caddy/caddy/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+Contributing to Caddy
```
Comment (nqb) [inline go/caddy/caddy/.github/CONTRIBUTING.md]: Is this file really necessary ?

File: go/caddy/caddy/.github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
```
Comment (nqb) [inline go/caddy/caddy/.github/ISSUE_TEMPLATE/bug_report.md]: Is this file really necessary ?

File: go/caddy/caddy/.github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (nqb) [inline go/caddy/caddy/.github/ISSUE_TEMPLATE/feature_request.md]: Is this file really necessary ?

File: go/caddy/caddy/.github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (nqb) [inline go/caddy/caddy/.github/PULL_REQUEST_TEMPLATE.md]: Is this file really necessary ?

File: go/caddy/caddy/azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,88 @@
+# Mutilated beyond recognition from the example at:
```
Comment (nqb) [inline go/caddy/caddy/azure-pipelines.yml]: Is this file really necessary ?

**Comment by fdurand** [comment]
Does it compile ?

**Comment by julsemaan** [comment]
It does on my build and I was able to login into the admin correctly + start all other pfhttpd based services
